timemachine/fe/interpolate.py

```diff
@@ -87,7 +87,7 @@ def make_kv(all_idxs, all_params):
 
         def has_duplicates(x):
             x = list(x)
-            return len(set(x)) < len(x)
+            return len(set(to_hashable(x))) < len(x)
```

Comment:
Was this change left in unintentionally? (tests still pass without it)

Reply:
reverted. This was my first attempt at a fix, and isn't needed anymore